### ISSUE TITLE
Error on deploy

### ISSUE THREAD

**Issue by gregmsanderson** [mnapoli/bref#47]
Given the message, this issue is with the underlying serverless framework, but having edited a single character in bref (related to the progress bar), deploys now all fail. It mentions a Process.php so is that the file that calls serverless to do the deploy on its behalf?

_Not_ very helpfully it does not show any error output, just complains about malformed XML. Given it happens at the end, the point where the file is uploaded to S3, it could be an issue with that? Strange. This is what happens, in case anyone else has had this. If not I can close and maybe take it up with serverless. Strange because I haven't changed the version of it and it was working earlier!

![deploy-error](https://user-images.githubusercontent.com/10390169/43232480-f58f6fa0-9068-11e8-9f15-18a5419c3204.jpg)

**Comment by gregmsanderson** [comment]
Yes, it does seem to be an issue with the underlying system used to deploy. Annoyingly nobody seems to have found a fix. Someone reported it with serverless but got no response: https://forum.serverless.com/t/cant-deploy-s3-complains-of-invalid-xml/3384

**Comment by gregmsanderson** [comment]
Interestingly I edited the serverless.yml file created by init to include some more details. And then the deploy works.

So I'm thinking one of these may be causing it given that was the only file I just changed, and it now works. Unless it is some random fault with the underlying SDK used to upload to S3, however that seems less likely.

So as an enhancement, as part of init, specify details like the region too. This is what I have and the deploy is (currently) working:

![edited-serverless-yml](https://user-images.githubusercontent.com/10390169/43279459-e8190050-9105-11e8-86a7-5099496430e4.jpg)

Not entirely sure how much of these bref uses. Or whether - since it uses serverless behind-the-scenes - it can use the same ones serverless does.
